docker/prepare_s3.sh

```diff
@@ -2,14 +2,14 @@
 
 # Check if voice-web bucket exists if not create it
 echo "Checking if s3 bucket exists..."
-status=$(curl -XGET s3proxy/common-voice-corpus --write-out "%{http_code}\n" --output /dev/null --silent)
+status=$(curl -XGET s3proxy/common-voice-clips --write-out "%{http_code}\n" --output /dev/null --silent)
```

Comment:
Instead of hardcoding bucket name here, should we take it from environment variable CV_CLIP_BUCKET_NAME  and set it in .env-local-docker ?

Reply:
`CV_CLIP_BUCKET_NAME` is an item in `.env-local-docker`, it's an environment variable.

Reply:
`CV_CLIP_BUCKET_NAME` is not an environment variable, it's an item in config file and not available to `docker/prepare_s3.sh`

Reply:
A possible solution is to put in in `.env`, and compose will pick it up as a environment variable.